DigitalLearningSolutions.Data.Tests/Services/DelegateApprovalsServiceTests.cs

```diff
@@ -0,0 +1,129 @@
+    using FakeItEasy;
+    using FluentAssertions;
+    using NUnit.Framework;
+
+    public class DelegateApprovalsServiceTests
+    {
+        private IUserDataService userDataService = null!;
+        private ICustomPromptsService customPromptsService = null!;
+        private IEmailService emailService = null!;
+        private ILogger<DelegateApprovalsService> logger = null!;
+        private IDelegateApprovalsService delegateApprovalsService = null!;
+
+        [SetUp]
+        public void SetUp()
+        {
+            userDataService = A.Fake<IUserDataService>();
+            customPromptsService = A.Fake<ICustomPromptsService>();
+            emailService = A.Fake<IEmailService>();
+            logger = A.Fake<ILogger<DelegateApprovalsService>>();
+            delegateApprovalsService = new DelegateApprovalsService(userDataService, customPromptsService, emailService, logger);
+        }
+
+        [Test]
+        public void GetUnapprovedDelegatesWithCustomPromptAnswersForCentre_returns_unapproved_delegates_with_custom_prompt_answers_for_centre()
+        {
+            // Given
+            var expectedDelegateUser = UserTestHelper.GetDefaultDelegateUser();
+            var expectedUserList = new List<DelegateUser> { expectedDelegateUser };
+            var expectedCustomPrompts = new List<CustomPromptWithAnswer>
+                { CustomPromptsTestHelper.GetDefaultCustomPromptWithAnswer(1, options: null, mandatory: true, answer: "answer") };
+
+            A.CallTo(() => userDataService.GetUnapprovedDelegateUsersByCentreId(2))
+                .Returns(expectedUserList);
+            A.CallTo(() => customPromptsService.GetCentreCustomPromptsWithAnswersByCentreIdForDelegateUsers(2, expectedUserList))
+                .Returns(new List<(DelegateUser delegateUser, List<CustomPromptWithAnswer> prompts)>{(expectedDelegateUser, expectedCustomPrompts)});
+
+            // When
+            var result = delegateApprovalsService.GetUnapprovedDelegatesWithCustomPromptAnswersForCentre(2);
+
+            // Then
+            result.Should().BeEquivalentTo(new List<(DelegateUser, List<CustomPromptWithAnswer>)> { (expectedDelegateUser, expectedCustomPrompts) });
+        }
+
+        [Test]
+        public void ApproveDelegate_approves_delegate()
+        {
+            // Given
+            var expectedDelegateUser = UserTestHelper.GetDefaultDelegateUser(approved: false);
+
+            A.CallTo(() => userDataService.GetDelegateUserByIdFromCentre(2, 2)).Returns(expectedDelegateUser);
+            A.CallTo(() => userDataService.ApproveDelegateUsers(A<IEnumerable<int>>.That.IsSameSequenceAs(new[] { 2 }))).DoesNothing();
+            A.CallTo(() => emailService.SendEmail(A<Email>._)).DoesNothing();
+
+            // When
+            delegateApprovalsService.ApproveDelegate(2, 2);
+
+            // Then
+            A.CallTo(() => userDataService.ApproveDelegateUsers(A<IEnumerable<int>>.That.IsSameSequenceAs(new[] { 2 }))).MustHaveHappened();

```

Comment:
Really minor, but I think IsSameSequenceAs can be just used with values, e.g.
`.IsSameSequenceAs(2) or IsSameSequenceAs(2, 3) `

This removes the extra syntax around creating an array :)